packages/suite-web/e2e/cypress.config.ts

```diff
@@ -1,6 +1,5 @@
 import { defineConfig } from 'cypress';
 
-// import CDP from 'chrome-remote-interface';
```

Comment:
hmm lines 41 and 176 are dead too. I think this was used to add hover ability to cypress. not sure why we commented this out? maybe @andrcj knows

Reply:
not sure if 176 is dead, `resetCRI` is used in `commands.ts`

Reply:
cc @andrcj I don't really maintain this anymore